src/components/App.js

```diff
@@ -1,13 +1,31 @@
+import { useState } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-// eslint-disable-next-line no-unused-vars
 import calculate from '../logic/calculate';
 
-const App = () => (
-  <>
-    <Display />
-    <ButtonPanel />
-  </>
-);
+const App = () => {
```

Comment:
- [ ] It should be something like `class App extends React.Component` and then proceed with the rest of the implementation.